x-pack/plugins/security_solution/public/common/components/charts/draggable_legend_item.test.tsx

```diff
@@ -68,4 +68,13 @@ describe('DraggableLegendItem', () => {
     );
     expect(wrapper.find('[data-test-subj="value-wrapper-empty"]').first().exists()).toBeTruthy();
   });
+
+  it('does not render the value label when the value is a number', () => {
```

Comment:
I would rename this `does not render the empty value label when...`.

Reply:
Ah yes, that's what it should've said in the first place 😅  https://github.com/elastic/kibana/pull/121274/commits/e78e1a06263f20530594693860bec61c6f5daca0